src/Rules/ConsoleCommand/NoConstructorDependencyInjectionRule.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Larastan\Larastan\Rules\ConsoleCommand;
+
+use Illuminate\Console\Command;
+use PhpParser\Node;
+use PhpParser\Node\Stmt\Class_;
+use PHPStan\Analyser\Scope;
+use PHPStan\Node\InClassNode;
+use PHPStan\Rules\Rule;
+use PHPStan\Rules\RuleError;
+use PHPStan\Rules\RuleErrorBuilder;
+
+use function count;
+use function sprintf;
+
+/** @implements Rule<InClassNode> */
+final class NoConstructorDependencyInjectionRule implements Rule
+{
+    public function getNodeType(): string
+    {
+        return InClassNode::class;
```

Comment:
Why don't we hook into `InClassMethod` node? And then we can also do something like
```php
$method = $node->getMethodReflection();

if (!$method->isConstructor()) {
	return [];
}
```

in `processNode` method.